cpp/segfault_handler.h

```diff
@@ -3,7 +3,7 @@
 
 static int segfault_handler_initialized;
 
-#define WRITE_STRING_TO_STDERR(s) (write(2, (s), sizeof(s) - 1))
+#define WRITE_STRING_TO_STDERR(s) do {size_t ignored = write(2, (s), sizeof(s) - 1);} while (false)
```

Comment:
Can this be static_cast<void*> instead?

https://stackoverflow.com/questions/689677/why-cast-unused-return-values-to-void

Reply:
It's not `void*`, it's `void`, and I tried that, and it didn't work